pkg/controller/template/render.go

```diff
@@ -332,6 +334,13 @@ func skipMissing(key string) (interface{}, error) {
 
 func cloudProvider(cfg RenderConfig) (interface{}, error) {
 	if cfg.Infra.Status.PlatformStatus != nil {
+		external, err := cloudprovider.IsCloudProviderExternal(cfg.Infra.Status.Platform, cfg.FeatureGate) //nolint
```

Comment:
Because of the deprecation warning. library-go still uses infra.status.platform, while is recommended to use infra.status.platformStatus.type now. This causes `verify` check to fail.

Reply:
double checking: is this still true?

Reply:
I guess not, we could use platformStatus.type, let me update this quickly. MCO does bootstrap part after cluster-config-operator, so it should work.

Reply:
PlatformStatus.Type is default from OCP 4.2 so I'd expect it to be the one we use here anyway